test/safe/branch.test.ts

```diff
@@ -160,14 +160,17 @@ describe("Branch Webhook", () => {
 
 			await expect(app.receive(fixture)).toResolve();
 			expect(parseSmartCommit).not.toBeCalled();
+
+			await waitUntil(async () => {
+				expect(githubNock).toBeDone();
+				expect(jiraNock).toBeDone();
+			});
 		});
 	});
 
 	describe("Create Branch (with disabled FF - delete this test with FF cleanup)", () => {
 		it("should update Jira issue with link to a branch on GitHub", async () => {
 
-			mockGitHubAuthRequest();
```

Comment:
It is flaky because you removed `mockGitHubAuthRequest();` from here

Reply:
if this is flaky, it means the code is flaky as well, having potentially 2 separate code paths for the same event.  If this is not an optional nock (which it shouldn't be), this test will fail as there's an extra nock that never gets called (the auth request). 

The flakiness can also be attributed to the `waitUntil` if it takes longer than the test timeout.